lib/tokenizers/tokenizer.ex

```diff
@@ -62,7 +62,7 @@ defmodule Tokenizers.Tokenizer do
   @spec encode(Tokenizer.t(), encode_input() | [encode_input()], Keyword.t()) ::
           {:ok, Encoding.t() | [Encoding.t()]} | {:error, term()}
   def encode(tokenizer, input, opts \\ []) do
-    add_special_tokens = Keyword.get(opts, :add_special_tokens, false)
+    add_special_tokens = Keyword.get(opts, :add_special_tokens, true)
```

Comment:
Ah, we should also mention the default in the docs, I forgot :)

Reply:
Ah good point. And I need to switch `skip_special_tokens` to true in `decode`.